exams/signals_test.py

```diff
@@ -38,7 +38,11 @@ def setUpTestData(cls):
         with mute_signals(post_save):
             cls.profile = profile = ProfileFactory.create()
 
-        program, _ = create_program(past=True)
+        program, _ = create_program(
+            past=True,
+            exam_series_code="TestExam",
+            exam_module="SampleCourse"
+        )
```

Comment:
this change is because signal uses `exams.utils.authorize_for_exam` and we change few things there